packages/components/src/table/README.md

```diff
@@ -0,0 +1,251 @@
+TableCard
```

Comment:
Noting some of the examples have no background in case that is unintentional.

Reply:
Wrapped those ones in `<Card>`s: 93f4b15f.